docs/_manage/making-changes.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Making Integration Changes in Platform UI and CLI
+order: 6
+layout: post-toc
+---
+
+# Making Integration Changes in Platform UI and CLI
+
+Before making updates to your integration, it's important to consider the potential impact on user migration and existing Zaps. Ensuring your API and Zapier integration remains backwards compatible is crucial to avoid disruption to users. However, we acknowledge certain changes are sometimes necessary and avoidable. In such cases, consider the best practices for implementation.
+
+## Affects of Different Changes (Versioning Matrix)
+
+The matrix below illustrates the impact of different changes on promotions and migrations for public integration. Refer to our best practices to facilitate the upgrade process for you and your users.
+
+* Update: Making a change to an existing component
+* Replace: Deleting/deprecating an existing component and adding a new one in its place
+* Breaking Change: any modification to the integration which renders existing Zaps incompatible with the new version
+
+Several change scenarios are validated by the platform when you try to "Migrate" after a version promotion, but always be aware of the affects of any changes you make before you even begin implementing those changes.
+
+| **Integration Change** | **Add** | **Update** | **Replace** | **Delete/Deprecate** | **Validated by platform?** |
+| --- | --- | --- | --- | --- | --- |
+| **Authentication schemes** | **BREAKING CHANGE** | n/a | **BREAKING CHANGE** | n/a | Yes |
```

Comment:
What do you think about the following symbols:
* `✓` to replace "OK"
* `-` to replace "n/a"

Totally up to you, but it may help reduce visual clutter a bit! See this page for example: https://platform.zapier.com/quickstart/zapier-platform

Reply:
Love this suggestion! Minimal and consistent - ty.